azure-communication-ui/azure-communication-ui/src/main/java/com/azure/android/communication/ui/presentation/fragment/calling/banner/BannerViewModel.kt

```diff
@@ -23,6 +24,10 @@ internal class BannerViewModel {
         return shouldShowBannerStateFlow
     }
 
+    fun getLocalizationProvider(): LocalizationProvider {
```

Comment:
I think it is better to have public method getLocalizedString(stringId) instead of getLocalizationProvider.
View does not need to know about LocalizationProvider. So it will look like this:
viewModel.getLocalizedString( stringId) 
instead of
viewModel.getLocalizationProvider().getLocalizedString(context, stringId)

Reply:
From an MVVM architecture perspective, we should not need to expose LocalizationProvider from the ViewModels to the View layer. It can be used within the ViewModels to get the strings, then publish the strings as properties that are consumed by the View layer.

@jimchou-dev @JoshuaLai

Reply:
Yes, that is my point.

Reply:
@waynemo confirmed that on iOS we don't expose the localization provider outside of the view models. There was one instance of it in our participant list view controller. But he was old refactor code that was unused so we are making a PR to clean it up